scripts/index_congress_scores.py

```diff
@@ -202,54 +129,46 @@ def get_sen_id(state, name):
 			elif row_num == 2:
 				aclu_position = row
 			elif name != 'LEGEND:' and name != '' and name != 'Z-Vacant':
-
 				print(name)
 
-				if state_district in reps:
```

Comment:
@dphiffer this if check doesn't seem to be doing anything useful right? I removed it because we now no longer have a `reps` object

Reply:
Yeah that seems right. Basically it's asking the question: "does this House Seat exist in our data structure?" And since that data structure is going away it seems right to remove the check.